### DISCUSSION TITLE
Every other message requires resubmitting chat

### DISCUSSION THREAD

**Discussion by BennyMorton28** [danny-avila/LibreChat#2979] [Troubleshooting]
### What happened?

When I ask questions it seems to time out after like 3-4 seconds and then goes blank. and then I resubmit the question, and after a few tries it ends up getting it. Does anyone know what's happening and how to fix it? Tried rebooting the system (sudo reboot) and composing down and back up and still not able to get it to respond to each question without a few repeated questions. Thanks!

<img width="709" alt="Screenshot 2024-06-06 at 3 01 43 PM" src="https://github.com/danny-avila/LibreChat/assets/156146537/e3a88a2b-2130-4d17-b49d-3fcd38be927b">

### Steps to Reproduce

1. Ask one question (first one always works fine)
2. Ask a second question (always glitches after second question)

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is not reproducible so I moved it to a Troubleshooting discussion.

![image](https://github.com/danny-avila/LibreChat/assets/110412045/e13bd1cc-540b-46e5-9798-f9014801ebb5)


Also, without logs, it's hard to figure out exactly why this is happening for you.

**Reply by BennyMorton28** [reply]
Thanks Danny, your comment made me think to check the logs. I went into the logs and saw that it was a rate limit issue. Fixed it and now it's working great!!
